### PR TITLE
NO-JIRA: cleanup root and app OWNERS

### PR DESCRIPTION
**Pull request by flavianmissi**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
   - bparees
-  - mfojtik
-  - soltysh
   - csrwng
   - sayan-biswas
   - divyansh42
+  - flavianmissi
 approvers:
   - adambkaplan
   - bparees
```
Comment (deads2k) [inline OWNERS:9]: suggest removing Ben too.

File: OWNERS [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-  - mfojtik
-  - soltysh
   - sayan-biswas
+  - flavianmissi
```
Comment (deads2k) [inline OWNERS:11]: I suggest bringing this up in a sync call to see who is actually knowledgeable versus who has been assigned bug fixes and there is no longer someone knowledgeable to merge them.

Comment (flavianmissi) [reply OWNERS]: Yup, will do. Removed my own name for now.

**Comment by divyansh42** [comment]
Please feel free to remove my name(`divyansh42`) from the list of reviewers as well.
Thanks!

**Comment by sanchezl** [comment]
I would like to be added.

**Comment by flavianmissi** [comment]
/retitle NO-JIRA: cleanup root and app OWNERS

**Comment by flavianmissi** [comment]
I'm seeing hypershift conformance failing in other places as well. Plus, this PR changes couldn't possibly have broken it. 
@deads2k could you override `ci/prow/e2e-hypershift-conformance` ?

**Comment by flavianmissi** [comment]
https://github.com/openshift/release/pull/57716 will hopefully fix `e2e-hypershift-conformance` job.

**Comment by adambkaplan** [comment]
Adding a JIRA to this will make your life a whole lot easier when it comes time to cherrypick.

**Pull request merged**